content/en/news/releases/1.6.x/announcing-1.6.13/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Istio 1.6.13
+linktitle: 1.6.13
+subtitle: Patch Release
+description: Istio 1.6.13 patch release.
+publishdate: 2020-10-23
```

Comment:
Publish date is wrong. I think it's planned for the 27th?

Reply:
D'oh. Put in Friday's date. Let me fix that. Thanks.